frappe/public/js/frappe/model/model.js

```diff
@@ -3,6 +3,14 @@
 
 frappe.provide("frappe.model");
 
+frappe.eval_as_iife = function (js) {
+	// using optional chaining avoid collusion with local scope
+	// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#using_indirect_eval
+	eval?.(`(() => {
```

Comment:
```suggestion
	return eval?.(`(() => {
```

Reply:
I am not defining a new frappe method anymore.